### PR TITLE
Update gitlab_dashboard.json

### PR DESCRIPTION
**Pull request by dpacheconr**
update dashboard broken image link

### CODE REVIEW

**Comment by aswanson-nr** [comment]
Not stale

File: dashboards/gitlab/gitlab_dashboard.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
             "id": "viz.markdown"
           },
           "rawConfiguration": {
-            "text": "![drawing](https://1000logos.net/wp-content/uploads/2022/02/Gitlab-Logo.png)\n"
+            "text": "![drawing](https://1000logos.net/wp-content/uploads/2022/03/Gitlab-Logo-500x282.jpg)\n"
```
Comment (rahulbasu-nr) [inline dashboards/gitlab/gitlab_dashboard.json:22]: Please use the image source - https://newrelic.com/sites/default/files/quickstarts/images/icons/gitlab--logo.svg 
We already have the logo hosted at New Relic.

Comment (dpacheconr) [reply dashboards/gitlab/gitlab_dashboard.json]: Done

**Review by rahulbasu-nr — APPROVED** [review]
Looks good to me.

**Pull request merged**